src/hooks/useRewriteSource.ts

```diff
@@ -0,0 +1,13 @@
+import { getSourceUrl } from '../helpers/getSourceUrl';
+import { useRewrites } from '../hooks/useRewrites';
+
+export interface RewriteSourceProps {
+  path: string;
+  locale: string;
+}
+
+export function useRewriteSource({ path, locale }: RewriteSourceProps): string {
+  const rewrites = useRewrites();
+
+  return getSourceUrl({ rewrites, path, locale });
+}
```

Comment:
This hook does so little anymore that if might be good to think whether it's needed at all. OTOH, it isn't harmful either so that consideration could be left for later.

Reply:
Yes, I've only created it to avoid duplication, I'll be using it later in `<AlternateLinks />` component. Indeed, we can decide later on if it's really needed. Thanks.